src/services/Look/config.ts

```diff
@@ -0,0 +1,20 @@
+import { IncomingMessage } from 'http';
+
+import { baseConfig } from '../Api';
+
+import { ILookInputDTO } from '@src/interfaces';
+
+export const listConfig = (req?: IncomingMessage) =>
+  baseConfig(true, req).get('/super/looks/', {});
```

Comment:
```suggestion
  baseConfig(true, req).get('/looks/', {});
```

모든 API가 super로 대응된 것이 아니라, pickkone에서 사용하는 경로 그대로 사용하시면 됩니다.

Reply:
f399f14